CommunicationCallingSampleMauiApp/Platforms/Android/Composite.cs

```diff
@@ -164,11 +206,11 @@ public void Finalized()
 
             public void Handle(Java.Lang.Object eventArgs)
             {
-                if (eventArgs is CallCompositeCallStateEvent)
-                {
-                    var callState = eventArgs as CallCompositeCallStateEvent;
-                    Console.WriteLine(callState.Code.ToString());
-                }
+                //if (eventArgs is CallCompositeCallStateEvent)
+                //{
+                //    var callState = eventArgs as CallCompositeCallStateEvent;
+                //    Console.WriteLine(callState.Code.ToString());
+                //}
```

Comment:
Commented out these lines as 1.4.0 aar could not find CallCompositeCallStateEvent class.

Reply:
please find new name and update :)